src/main/webapp/app/overview/course-lectures/course-lectures.component.ts

```diff
@@ -37,6 +37,9 @@ export class CourseLecturesComponent implements OnInit, OnDestroy {
         private route: ActivatedRoute,
     ) {}
 
+    /**
+     * On init, gets the course and group its lectures
```

Comment:
I think it would be good to be more detailed in describing the way the lectures of the course are grouped. For instance, you could link the usage of the groupLectures method and also describe the setting of totalAttachmentCount by linking the method getAttachmentCount